CONTRIBUTORS

```diff
@@ -19,5 +19,5 @@ Stefano Rolfo (STFC, UK)
 Kay Schäfer (Karlsruhe, Germany)
 Felipe Schuch (Porto Alegre, Brazil)
 Filippo Spiga (NVIDIA, UK)
-
+Gaurav Gupta (IIST, India)
```

Comment:
The first line of the file clearly indicates the authors are in alphabetic order

Reply:
Sorry about that ! I missed it.